index.js

```diff
@@ -0,0 +1,86 @@
+const myLibrary = [];
+const target = document.getElementById('demo');
+const targetbtn = document.getElementById('add-btn');
+const targetdisplay = document.getElementById('display');
+const alertText = document.getElementById('alertText');
+let counter = 0;
+
+function Book(booktitle, author, totalPages, read) {
+  this.booktitle = booktitle;
+  this.author = author;
+  this.totalPages = totalPages;
+  this.read = read;
+}
+
+function saveLibrary() {
+  localStorage.setItem('myLibrary', JSON.stringify(myLibrary));
+}
+
+function addBookToLibrary(booktitle, author, totalPages, read = false) {
+  const book = new Book(booktitle, author, totalPages, read);
+  myLibrary.push(book);
+  saveLibrary();
+}
+
+function displayForm() {
+  targetbtn.style.display = 'none';
+  target.style.display = 'none';
+  targetdisplay.style.display = 'block';
+}
+
+function displayBooks() {
+  let count = 0;
+  function myFunction(obj) {
+    if (count >= counter) {
+      const newData = document.createElement('div');
```

Comment:
## Modularise this file
- [ ] Do you mind keeping `application` logic separated from `DOM manipulation` tasks as directed in this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/dom_logic_separation_requirement.png) from Pathwright?  
 
I recommend this because `separation of concerns` as it is known is the idea that each module or layer in an application should only be responsible for one thing and should not contain code that deals with other things. Separating concerns reduces code complexity by breaking a large application down into many smaller units of encapsulated functionality. Please feel free to read more about this suggestion [here](https://www.oreilly.com/library/view/programming-javascript-applications/9781491950289/ch05.html).